utils/githooks/install

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -eu
+
+githook_root="${0%/*}"
+repo_root="${githook_root}/../../"
+
+pushd "$repo_root" > /dev/null
+for d in "${githook_root}"/*.d; do
+    h=${d%.d}
+    h=${h##*/}
+    link_name=".git/hooks/$h"
+    link_target="../../utils/githooks/$h"
+    if [ -e "$link_name" ]; then
+        if [ "$(readlink ""$link_name"")" != "$link_target" ]; then
```

Comment:
(lint) The surrounding quotes actually unquote this. Remove or escape them. [SC2027]
(lint) Double quote to prevent globbing and word splitting. [SC2086]